code/game/objects/items/RCD.dm

```diff
@@ -536,12 +536,34 @@ GLOBAL_VAR_INIT(icon_holographic_window, init_holographic_window())
 	user.visible_message(span_suicide("[user] pulls the trigger... But there is not enough ammo!"))
 	return SHAME
 
+/**
+ * checks if the turf has no dense objects that could block construction of big structures such as walls, airlocks etc
+ * Arguments:
+ * * turf/target_turf - The turf we are checking
+ * * ignore_mobs - should we ignore mobs when checking for dense objects. this is FALSE only for windoors
+ * * ignored_atoms - ignore these object types when checking for dense objects on the turf. ege. ignore directional windows when building windoors cause they all can exist on the same turf
+ */
+/obj/item/construction/rcd/proc/density_check(turf/target_turf, ignore_mobs, list/ignored_atoms)
```

Comment:
Name of this is confusing, and the documentation doesn't help. Feels like TRUE means it *is* dense